the-depot/Services/ReservationService.cs

```diff
@@ -109,12 +109,18 @@ public static void ScanCodeToAttend(Tour tour, string message)
             addResponse = "";
             var cancelled = CancelReservation(dayKey_Id, out string cancelResponse);
             addResponse = cancelResponse;
+            var tour = TourRepository.Get(tour_Id);
+            var attendeesCount = ReservationRepository.FindByTour(tour_Id).Count();
+            if (tour.MaxAttendees <= attendeesCount)
```

Comment:
warning fixen, hij mag wel stuk gaan als tour null is want dat hoort nooit te gebeuren. Suggestie: "tour!"